wolfssl/wolfcrypt/types.h

```diff
@@ -148,7 +149,10 @@
 
 	/* set up thread local storage if available */
 	#ifdef HAVE_THREAD_LS
-	    #if defined(_MSC_VER)
+        #if defined(INTIME_RTOS)
+            /* Thread local storage not supported */
+            #define THREAD_LS_T
```

Comment:
If not actually supported won't this break use cases that assume the type is valid?  Why is HAVE_THREAD_LS defined if thread local storage is not available?